website/views.py

```diff
@@ -142,26 +143,29 @@ def index():
 
     user_id = get_current_user_id()
     if user_id:  # Logged in: return either landing page or user home page
-        all_institutions = (
-            Institution.objects.filter(
-                is_deleted=False,
-                nodes__is_public=True,
-                nodes__is_deleted=False,
-                nodes__type='osf.node'
+        if waffle.switch_is_active('ember_dashboard'):
+            return send_from_directory(ember_osf_web_dir, 'index.html')
```

Comment:
Will this work with the ember proxied server?

Reply:
This won't work in staging or prod. It must be proxied like in https://github.com/CenterForOpenScience/osf.io/blob/develop/website/ember_osf_web/views.py

Is that a better place for this to go?